metasv/generate_sv_intervals.py

```diff
@@ -25,6 +26,22 @@ def concatenate_files(files, output):
                 outfile.write(infile.read())
 
 
+def find_softclip(aln):
+    if aln.cigar is None:
+        return None
+    soft_clips = [(i, length) for i,(op, length) in enumerate(aln.cigar) if op == 4]
+    if len(soft_clips) != 1:
+        return None
+    
+    i, soft_clip = soft_clips[0]
+    dist_L_end = sum(map(lambda x:x[1], aln.cigar[0:i]))
```

Comment:
Are you treating deletions and insertions in the CIGAR string the same way? If the distance is in the read, then deletions must be ignored.